virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -36,9 +36,13 @@ include::modules/virt-supported-tekton-tasks.adoc[leveloffset=+1]
 
 The Tekton Tasks Operator includes the following example `Pipeline` manifests. You can run the example pipelines by using the web console or CLI.
 
-Windows 10 installer pipeline:: This pipeline installs Windows 10 into a new data volume from a Windows installation image (ISO file). A custom answer file is used to run the installation process.
+You may need to run more than one installer pipline if you need multiple versions of Windows. If you run more than one installer pipeline, each one requires unique parameters, such as the `autounattend`` config map and base image name. For example, if you need Windows 10 and Windows 11 or Windows Server 2022 images, you will need to run both the Windows efi installer pipeline and the Windows bios installer pipeline. However, if you need Windows 11 and Windows Server 2022 images, you only need to run the Windows efi installer pipeline.
 
-Windows 10 customize pipeline:: This pipeline clones the data volume of a basic Windows 10 installation, customizes it by installing Microsoft SQL Server Express, and then creates a new image and template.
+Windows efi installer pipeline:: This pipeline installs Windows 11 or Windows Server 2022 into a new data volume from a Windows installation image (ISO file). A custom answer file is used to run the installation process.
+
+Windows bios installer pipeline:: This pipeline installs Windows 10 into a new data volume from a Windows installation image (ISO file). A custom answer file is used to run the installation process.
+
+Windows customize pipeline:: This pipeline clones the data volume of a basic Windows 10, 11, or Windows Server 2022 installation, customizes it by installing Microsoft SQL Server Express, and then creates a new image and template.
```

Comment:
Windows customize file also use a custom answer file. @ksimon1 should we add it?
Also do you think @ksimon1 we should add that for now we don't support any other versions especially with product key need or if we allow what needs to be done?

Reply:
@ksimon1 

> Currently we have two different xmls for customize pipeline - windows-sqlserver and windows-vs-code. The windows-sqlserver configmap installs Microsoft SQL Server Express, the windows-vs-code configmap installs Visual Studio Code.

How does the user specify the different XMLs for SQL Server or VSCode?

Reply:
There is a special pipeline parameter `customizeConfigMapName`, with it user can change which xml will be used in pipeline.
About the example of xml configmap: https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows-customize-configmaps.yaml#L2

Reply:
@sabrinajess Do you have time to turn the configmap example into a reference module? If not, I can break out the task into a separate Jira, which would have to be done post-GA.

Reply:
I don't think it is a good idea to insert a different xml example into documentation, because any updates to our examples are not supported

Reply:
@ksimon1 In that case, can we use a generic example like this?

---
apiVersion: v1
kind: ConfigMap
metadata:
  name: windows-sqlserver | windows-vs-code
data:
  unattend.xml: |
  <xml_file>
  ...
  ---